### ISSUE TITLE
Merchant crashes game

### ISSUE THREAD

**Issue by WindsongL** [apace100/origins-classes#43]
I wanted to test out merchant in a single player world before I chose it on a server. Whenever I trade more than the max vanilla amount the game freezes, mobs stop moving and the game gets stuck on 'saving world' when trying to exit.

**Comment by apace100** [comment]
Were you by chance playing as an Arachnid, and on Origins 1.0.0? There was a bug with Arachnids where the world froze.

**Comment by WindsongL** [comment]
No, I was on the most recent version. I was originally avian (as that’s what I play as), but I tried it as a human as well and it still crashed.
